src/employees.js

```diff
@@ -48,9 +48,9 @@ export const EmployeeList = () => (
 );
 
 export const EmployeeEdit = () => (
-    <EditEntityLayout inputsList={inputsList} referenceValues={referenceValues} />
+    <EditEntityLayout inputsList={inputsList} referenceValues={referenceValues} selectItem={false} />
```

Comment:
Instead of pass the selectItem param here, i would add it in the referenceValues object as:
```
const referenceValues = {
    source: 'profile',
    reference: 'roles',
    optionText: 'name',
   multiselect: true or false
}
```

then in the component:
`referenceValues && !referenceValues.multiselect`
and
`referenceValues && referenceValues.multiselect`

Reply:
great!